lightning/src/util/events.rs

```diff
@@ -172,6 +205,14 @@ pub enum Event {
 		/// transaction.
 		claim_from_onchain_tx: bool,
 	},
+	/// Used to indicate that a channel with the given `channel_id` is in the process of closure.
+	ChannelClosed  {
+		/// The channel_id which has been barren from further off-chain updates but
+		/// funding output might still be not resolved yet.
+		channel_id: [u8; 32],
+		/// A machine-readable error message
```

Comment:
This isn't an "error message", maybe simply "The reason the channel was closed"? Also its strange to call the varaible `err`

Reply:
Rename it to `reason`.